ExchangeSharp/Utility/CryptoUtility.cs

```diff
@@ -149,6 +149,35 @@ public static SecureString StringToSecureString(string unsecure)
             return secure;
         }
 
+        public static decimal ClampQuantity(decimal minQuantity, decimal maxQuantity, decimal? stepSize, decimal quantity)
+        {
+            if (stepSize.HasValue)
+            {
+                quantity = Math.Min(maxQuantity, quantity);
+                quantity = Math.Max(minQuantity, quantity);
+                quantity -= quantity % stepSize.Value;
+                quantity = Floor(quantity);
+            }
+            return quantity;
+        }
+
+        public static decimal ClampPrice(decimal minPrice, decimal maxPrice, decimal? tickSize, decimal price)
```

Comment:
How about adding unit tests for these methods to prove that they're working exactly as you think and prevent others from breaking them?